src/app/containers/ArticleMain/index.jsx

```diff
@@ -17,15 +18,23 @@ const componentsToRender = {
   timestamp,
 };
 
+const avEnabledComment = (
+  // eslint-disable-next-line react/no-danger
+  <div dangerouslySetInnerHTML={{ __html: '<!-- av-enabled -->' }} />
+);
+
 const ArticleMain = ({ articleData }) => {
   const { content, metadata, promo } = articleData;
   const { blocks } = content.model;
 
+  const { audiovideo: audioVideoEnabled } = useContext(DialContext);
+
   return (
     <Fragment>
       <ATIAnalytics data={articleData} />
       <MetadataContainer metadata={metadata} promo={promo} />
       <main role="main">
+        {audioVideoEnabled && avEnabledComment}
```

Comment:
Are we sure the value in the Cosmos config for dials will come through as a boolean instead of as a string? `"false"` vs `false` will have very different results.

Reply:
The implementation of `getDials` promotes `"true"` and `"false"` to booleans so I think it's saf (and seems to work in practice), see https://github.com/bbc/simorgh/blob/46c3c6c489bc06d2cc37f7db9db12e2be280d6fd/src/server/getDials/index.js#L3